x-pack/plugins/monitoring/server/lib/elasticsearch/shards/get_shard_stats.js

```diff
@@ -58,6 +58,8 @@ export function getShardStats(req, esIndexPattern, cluster, { includeNodes = fal
     }
   };
 
+  console.log('CHRIS', JSON.stringify(params));
+
```

Comment:
please remove :D

Reply:
haha oh wow I'm ashamed that I left that in!

Reply:
Eh, we've all done it. https://github.com/elastic/kibana/pull/21002 :D